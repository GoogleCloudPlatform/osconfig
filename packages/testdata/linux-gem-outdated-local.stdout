benchmark (0.1.0 < 0.4.0)
bigdecimal (2.0.0 < 3.1.9)
bundler (2.1.2 < 2.6.8)
cgi (0.1.0 < 0.4.2)
csv (3.1.2 < 3.3.4)
date (3.0.0 < 3.4.1)
delegate (0.1.0 < 0.4.0)
did_you_mean (1.4.0 < 2.0.0)
etc (1.1.0 < 1.4.5)
fcntl (1.0.0 < 1.2.0)
fiddle (1.0.0 < 1.1.8)
fileutils (1.4.1 < 1.7.3)
forwardable (1.3.1 < 1.3.3)
getoptlong (0.1.0 < 0.2.1)
io-console (0.5.3 < 0.8.0)
ipaddr (1.2.2 < 1.2.7)
irb (1.2.1 < 1.15.2)
json (2.3.0 < 2.11.3)
logger (1.4.2 < 1.7.0)
matrix (0.2.0 < 0.4.2)
minitest (5.13.0 < 5.25.5)
mutex_m (0.1.0 < 0.3.0)
net-pop (0.1.0 < 0.1.2)
net-smtp (0.1.0 < 0.5.1)
net-telnet (0.1.1 < 0.2.0)
observer (0.1.0 < 0.1.2)
open3 (0.1.0 < 0.2.1)
openssl (2.1.2 < 3.3.0)
ostruct (0.2.0 < 0.6.1)
power_assert (1.1.7 < 2.0.5)
prime (0.1.1 < 0.1.3)
pstore (0.1.0 < 0.2.0)
psych (3.1.0 < 5.2.4)
racc (1.4.16 < 1.8.1)
rake (13.0.1 < 13.2.1)
rdoc (6.2.1 < 6.13.1)
readline (0.0.2 < 0.0.4)
readline-ext (0.1.0 < 0.2.0)
reline (0.1.2 < 0.6.1)
rexml (3.2.3 < 3.4.1)
rss (0.2.8 < 0.3.1)
singleton (0.1.0 < 0.3.0)
stringio (0.1.0 < 3.1.7)
strscan (1.0.3 < 3.1.5)
test-unit (3.3.5 < 3.6.8)
timeout (0.1.0 < 0.4.3)
tracer (0.1.0 < 0.2.3)
uri (0.10.0 < 1.0.3)
webrick (1.6.0 < 1.9.1)
xmlrpc (0.3.0 < 0.3.3)
yaml (0.1.0 < 0.4.0)
zlib (1.1.0 < 3.2.1)
