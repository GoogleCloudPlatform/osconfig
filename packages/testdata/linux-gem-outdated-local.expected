[]*packages.PkgInfo{
    &packages.PkgInfo{
        Name:    "benchmark",
        Arch:    "all",
        RawArch: "",
        Version: "0.4.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "bigdecimal",
        Arch:    "all",
        RawArch: "",
        Version: "3.1.9",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "bundler",
        Arch:    "all",
        RawArch: "",
        Version: "2.6.8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "cgi",
        Arch:    "all",
        RawArch: "",
        Version: "0.4.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "csv",
        Arch:    "all",
        RawArch: "",
        Version: "3.3.4",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "date",
        Arch:    "all",
        RawArch: "",
        Version: "3.4.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "delegate",
        Arch:    "all",
        RawArch: "",
        Version: "0.4.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "did_you_mean",
        Arch:    "all",
        RawArch: "",
        Version: "2.0.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "etc",
        Arch:    "all",
        RawArch: "",
        Version: "1.4.5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "fcntl",
        Arch:    "all",
        RawArch: "",
        Version: "1.2.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "fiddle",
        Arch:    "all",
        RawArch: "",
        Version: "1.1.8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "fileutils",
        Arch:    "all",
        RawArch: "",
        Version: "1.7.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "forwardable",
        Arch:    "all",
        RawArch: "",
        Version: "1.3.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "getoptlong",
        Arch:    "all",
        RawArch: "",
        Version: "0.2.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "io-console",
        Arch:    "all",
        RawArch: "",
        Version: "0.8.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "ipaddr",
        Arch:    "all",
        RawArch: "",
        Version: "1.2.7",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "irb",
        Arch:    "all",
        RawArch: "",
        Version: "1.15.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "json",
        Arch:    "all",
        RawArch: "",
        Version: "2.11.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "logger",
        Arch:    "all",
        RawArch: "",
        Version: "1.7.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "matrix",
        Arch:    "all",
        RawArch: "",
        Version: "0.4.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "minitest",
        Arch:    "all",
        RawArch: "",
        Version: "5.25.5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "mutex_m",
        Arch:    "all",
        RawArch: "",
        Version: "0.3.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "net-pop",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "net-smtp",
        Arch:    "all",
        RawArch: "",
        Version: "0.5.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "net-telnet",
        Arch:    "all",
        RawArch: "",
        Version: "0.2.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "observer",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "open3",
        Arch:    "all",
        RawArch: "",
        Version: "0.2.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "openssl",
        Arch:    "all",
        RawArch: "",
        Version: "3.3.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "ostruct",
        Arch:    "all",
        RawArch: "",
        Version: "0.6.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "power_assert",
        Arch:    "all",
        RawArch: "",
        Version: "2.0.5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "prime",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "pstore",
        Arch:    "all",
        RawArch: "",
        Version: "0.2.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "psych",
        Arch:    "all",
        RawArch: "",
        Version: "5.2.4",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "racc",
        Arch:    "all",
        RawArch: "",
        Version: "1.8.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "rake",
        Arch:    "all",
        RawArch: "",
        Version: "13.2.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "rdoc",
        Arch:    "all",
        RawArch: "",
        Version: "6.13.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "readline",
        Arch:    "all",
        RawArch: "",
        Version: "0.0.4",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "readline-ext",
        Arch:    "all",
        RawArch: "",
        Version: "0.2.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "reline",
        Arch:    "all",
        RawArch: "",
        Version: "0.6.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "rexml",
        Arch:    "all",
        RawArch: "",
        Version: "3.4.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "rss",
        Arch:    "all",
        RawArch: "",
        Version: "0.3.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "singleton",
        Arch:    "all",
        RawArch: "",
        Version: "0.3.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "stringio",
        Arch:    "all",
        RawArch: "",
        Version: "3.1.7",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "strscan",
        Arch:    "all",
        RawArch: "",
        Version: "3.1.5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "test-unit",
        Arch:    "all",
        RawArch: "",
        Version: "3.6.8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "timeout",
        Arch:    "all",
        RawArch: "",
        Version: "0.4.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "tracer",
        Arch:    "all",
        RawArch: "",
        Version: "0.2.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "uri",
        Arch:    "all",
        RawArch: "",
        Version: "1.0.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "webrick",
        Arch:    "all",
        RawArch: "",
        Version: "1.9.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "xmlrpc",
        Arch:    "all",
        RawArch: "",
        Version: "0.3.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "yaml",
        Arch:    "all",
        RawArch: "",
        Version: "0.4.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "zlib",
        Arch:    "all",
        RawArch: "",
        Version: "3.2.1",
        Source:  packages.Source{},
    },
}